packages/rich-text/src/plugins/Hyperlink/__tests__/createHyperlinkPlugin.test.tsx

```diff
@@ -0,0 +1,39 @@
+/** @jsx jsx */
+import { jsx, assertOutput } from '../../../test-utils';
+
+describe('normalization', () => {
+  it('removes empty links from the document structure', () => {
+    const input = (
+      <editor>
+        <hp>
+          <htext>link</htext>
+          <hlink uri="https://link.com" />
```

Comment:
Can we add a link here with a value of just whitespaces, explicitly? e.g. ` <hlink uri="https://link.com">{'    '}</hlink>`. I not sure if these will be trimmed automatically on comparison in the `isAncestorEmpty` helper.

Reply:
Sure, I will try. Right now I can't add a link when selecting a text.